src/derivative_operators/derivative_operator_functions.jl

```diff
@@ -106,3 +106,80 @@ function *(c::Number, A::DerivativeOperator{T,N,Wind}) where {T,N,Wind}
         A.low_boundary_coefs,
         A.high_boundary_coefs,coefficients,A.coeff_func)
 end
+
+# Inplace left division
+function LinearAlgebra.ldiv!(M_temp::AbstractArray{T,MT}, A::DerivativeOperator{T,N}, M::AbstractArray{T,MT}) where {T<:Real, N, MT}
+
+    # The case where M is a vector or matrix and A is differentiating along the first dimension
+    if N == 1 && MT <= 2
+        ldiv!(M_temp, factorize(Array(A)), M)
+
+    # The case where M is differentiating along an arbitrary dimension
+    else
+        Mshape = size(M)
+
+        # Case where the first dimension is not being differentiated
+        if N != 1
+
+            # Compute the high dimensional concretization B of A
+
+            B = Matrix(I, Mshape[1],Mshape[1])
+            for i in length(Mshape)-1:-1:1
+                if N != length(Mshape) - i + 1
+                    B = Kron(Matrix(I,Mshape[i],Mshape[i]),B)
+                else
+                    B = Kron(Array(A),B)
+                end
+            end
+
+        # Case where the first dimension is being differentiated
+        else
+            B = Array(A)
+            for i in len(Mshape)-1:1
+                B = Kron(Matrix(I,Mshape[i],Mshape[i]),B)
+            end
+        end
+
+        # compute ldiv!
+        ldiv!(vec(M_temp), factorize(Array(B)), vec(M))
+    end
+end
+
+# Non-inplace left division.
+function \(A::DerivativeOperator{T,N}, M::AbstractArray{T,MT}) where {T<:Real, N,MT}
```

Comment:
this can just allocate and call the first?

Reply:
I don't think there's no a `ldiv!` implementation for sparse matrices in general (according to what I could find in `SparseArrays.jl`). So from my understanding, if we want to exploit the sparse structure of our operators we should be using `\`.